### PR TITLE
Bug squash

### PR DESCRIPTION
**Pull request by kfoley123**

### CODE REVIEW

File: public/_redirects [hunk 1]
```diff
@@ -0,0 +1 @@
+/*    /index.html   200
```
Comment (oakeef) [inline public/_redirects:1]: this file needs a newline at the end

File: src/Components/ListenOn/ListenOn.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ export default function ListenOn(props) {
 
                 {listenLinks}
             </div>
-            <Link className="navLink" to="/About">
```
Comment (oakeef) [inline src/Components/ListenOn/ListenOn.js:49]: if you no longer need this class you can also remove any stlying that went with it as well

File: src/Components/Patreon/Patreon.css [hunk 3]
```diff
@@ -19,13 +19,13 @@
     display: flex;
     flex-direction: column;
     max-width: 600px;
-    padding-left: 100px;
+    padding: 70px 0 0 100px;
```
Comment (oakeef) [inline src/Components/Patreon/Patreon.css:22]: good optimization here!

File: src/Pages/OtherProjects/OtherProjects.css [hunk 4]
```diff
@@ -35,6 +35,16 @@
     height: 300px;
 }
 
+.TheLastOfUsEpContainer button.carouselButton:nth-child(3) + div {
```
Comment (oakeef) [inline src/Pages/OtherProjects/OtherProjects.css:38]: i'm curious if you could target both carousels with just `button.carouselButton:nth-child(3) + div` ??

Comment (kfoley123) [reply src/Pages/OtherProjects/OtherProjects.css:38]: looks like it needs the ep container as well  to work

Comment (oakeef) [reply src/Pages/OtherProjects/OtherProjects.css:38]: I thought it might. Wasn't sure though. Sounds good

**Review by oakeef — APPROVED** [review]

**Pull request merged**